web/app/js/components/NamespaceLanding.jsx

```diff
@@ -213,4 +227,4 @@ class NamespaceLanding extends React.Component {
   }
 }
 
-export default withContext(NamespaceLanding);
+export default withContext(withStyles(styles, { withTheme: true })(NamespaceLanding));
```

Comment:
we're including `withTheme: true` here but we don't use the theme in our styles function, so this config can be omitted.